This patch REVERTS the commit below

commit c18f3d25a94eb72145c663797794d2697a226594
Author: Neil Horman <nhorman@gmail.com>
Date:   Mon Mar 15 08:26:30 2021 -0400

    Update configure to fail if readline isn't installed
    
    We had libreadline pkg check in configure hacked a bit.  Because
    libreadline didn't have a .pc file for years, we fell back on just
    adding the library as a linker option and letting the compile fail when
    readline.h couldn't be found.  Now that readline 8.0 has been out for 2
    years (which included the .pc file), I think we can safely test for it,
    and fail if its not found
    
    Signed-off-by: Neil Horman <nhorman@tuxdriver.com>

diff --git a/configure.ac b/configure.ac
index 278da54..2a840cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -16,6 +16,6 @@ AC_CHECK_FUNCS(getopt_long)
 PKG_CHECK_MODULES([LIBNL3], [libnl-3.0], [], [AC_MSG_ERROR([libnl-3.0 is required])])
 PKG_CHECK_MODULES([LIBNLG3], [libnl-genl-3.0], [], [AC_MSG_ERROR([libnl-genl-3.0 is required])])
+PKG_CHECK_MODULES([READLINE], [readline], [], [READLINE_LIBS=-lreadline])
-PKG_CHECK_MODULES([READLINE], [readline], [], [AC_MSG_ERROR([libreadline is required])])
 PKG_CHECK_MODULES([LIBPCAP], [libpcap], [], [
         AC_CHECK_LIB(pcap, pcap_open_live,[],
                 [AC_MSG_ERROR([libpcap is required])])])
